### PR TITLE
SetStakingAuthority instruction removed

### PR DESCRIPTION
**Pull request by ysavchenko**

### CODE REVIEW

**Comment by joncinque** [comment]
This looks good to me.  I'm going to hold off on merging until we get another opinion.

**Review by mvines — APPROVED** [review]

**Comment by ysavchenko** [comment]
Let's merge! @joncinque

**Comment by joncinque** [comment]
@ysavchenko Could you rebase your branch?  There was a small change in the `set-staking-authority.rs` which isn't consequential since the file is getting deleted anyway :-)

**Pull request closed without merging**
